### DISCUSSION TITLE
We now have Discussion Types!

### DISCUSSION THREAD

**Discussion by tkralphs** [coin-or/Cbc#335] [Show and tell]
Now when you go to https://github.com/coin-or/Cbc/discussions, you'll notice that you have the ability to specify a "type" for your Discussion. This looks great both for giving people an idea about the kinds of things they can share here and also organizing and categorizing the discussions. Please give a try!

**Comment by svigerske** [comment]
TIL is "Today I learned"? Would be nice to write this out, but I didn't see how to change this.

**Reply by tkralphs** [reply]
Right now, the categories are hard-coded, but by launch, we'll be able to customize them.

**Reply by tkralphs** [reply]
Also, if you click on "New Discussion," it lists the topics in more detail and spells out "Today I Learned."
